recipe/meta.yaml

```diff
@@ -101,8 +101,9 @@ about:
 
 extra:
   recipe-maintainers:
+    - jan-janssen
     - dalcinl
     - minrk
     - davidbrochart
     - SylvainCorlay
-    - martinRenou
+    - martinRenou
```

Comment:
Please fix this, you removed the newline at end of file.

Reply:
This is an automated pull request which was triggered by opening the issue https://github.com/conda-forge/petsc-feedstock/issues/129 . So I am convinced that removing the newline at the end does not change anything. Unfortunatley I do not have access to the pull requests by `conda-forge-linter` but you should have access and could add the new line again.